tgstation.dme

```diff
@@ -2268,7 +2268,6 @@
 #include "code\modules\antagonists\traitor\objectives\destroy_item.dm"
 #include "code\modules\antagonists\traitor\objectives\destroy_machinery.dm"
 #include "code\modules\antagonists\traitor\objectives\hack_comm_console.dm"
-#include "code\modules\antagonists\traitor\objectives\kidnapping.dm"
```

Comment:
Ah I see the solution was to disinclude the tg file instead and place our own local modified copy, that also seems reasonable actually.

Reply:
This means our work won't cause issues, but if the base files change, but it will be also very annoying to copy over updates to our branch.